assets/bingo/lang/en_us.json

```diff
@@ -27,10 +27,15 @@
 	"bingo.logging.error": "ERROR",
 	"bingo.setup.error.unknown_category": "Item %1$s has category %2$s, but category %2$s could not be found.",
 
-	"bingo.game.go_to_lobby": "%s to go back to the lobby.\n%s to switch to spectator mode.",
-	"bingo.game.go_to_lobby.hint": "Hint: You can also use the command %s at any time manually to return to the lobby or %s for spectator mode.",
-	"bingo.game.go_to_lobby.click_lobby": "Click here",
-	"bingo.game.go_to_lobby.click_spectator": "Click here",
+	"bingo.game.menu.text": "[%s]",
+	"bingo.game.menu.icon": "≡",
+	"bingo.game.menu.title": "Bingo Menu",
+	"bingo.game.menu.hover_text": "Bingo Menu",
+	"bingo.game.menu.hint": "Hint: You can click %s or use the command %s at any time to display the bingo menu.",
+	"bingo.game.menu.options": "%s to go back to the lobby.\n%s to switch to spectator mode.",
+	"bingo.game.menu.options.click_lobby": "Click here",
+	"bingo.game.menu.options.click_spectator": "Click here",
+	"bingo.game.menu.options.hint": "Hint: You can also use the command %s at any time manually to return to the lobby or %s for spectator mode.",
```

Comment:
To achieve that, I think a new translation would be needed for the part in parentheses since that section needs to be a separate colour/italicised and therefore needs to be separate from the rest of the message.
```
"bingo.game.menu.options": "%s to go back to the lobby (%s)\n%s to switch to spectator mode (%s)",
"bingo.game.menu.options.hint": "Or run %s",
```

And so would this replace the original hint message?
`Hint: You can also use the command /trigger bingo.lobby at any time manually to return to the lobby or /trigger bingo.spectator for spectator mode.`

Reply:
> To achieve that, I think a new translation would be needed for the part in parentheses since that section needs to be a separate colour/italicised and therefore needs to be separate from the rest of the message.
> 
> ```
> "bingo.game.menu.options": "%s to go back to the lobby (%s)\n%s to switch to spectator mode (%s)",
> "bingo.game.menu.options.hint": "Or run %s",
> ```

Yes

> And so would this replace the original hint message?
> `Hint: You can also use the command /trigger bingo.lobby at any time manually to return to the lobby or /trigger bingo.spectator for spectator mode.`

Yes

> I think it would make things a lot neater to just have the "hint" always next to the options in parentheses instead of as a whole separate message like it was before.

I agree ;)

Reply:
I suppose the best solution might be to merge them into a single hint, for example like this:
![grafik](https://user-images.githubusercontent.com/12124394/171252433-58fb8ab8-1eae-4f58-8b6a-b719b43c9ea4.png)

And not have any separate hint for the commands.